source/agora/utils/WellKnownKeyData.d

```diff
@@ -0,0 +1,7041 @@
+/*******************************************************************************
+
+    The list of well-known keypairs
+
+    This module should not be imported directly.
+    Instead use `agora.utils.Test : WK.Keys`.
+    It is solely here to host all well-known keys in a separate, single file,
+    as they are stored as array of `ubyte` to speed-up compilation.
+
+    Copyright:
+        Copyright (c) 2019-2021 BOSAGORA Foundation
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+module agora.utils.WellKnownKeyData;
```

Comment:
Wait why a new module ? We already have one for this ?

Reply:
Actually we need this because `agora.crypto.Key` module can not refer to `agora.utils.WellKnownKeys`. I created the module `agora.utils.WellKnownHRPs' module and made the module more compact anyway.